src/tools/miri/src/bin/log/tracing_chrome.rs

```diff
@@ -526,16 +523,16 @@ where
                 }
             },
             TraceStyle::Async => Some(
-                span.scope()
-                    .from_root()
-                    .take(1)
-                    .next()
-                    .unwrap_or(span)
-                    .id()
-                    .into_u64()
+        span.scope()
```

Comment:
Uh, what is happening here?

This PR has introduced a bunch of nonsensical whitespace changes across various Miri files -- none of which rustfmt is able to fix, apparently.

Reply:
Can you please submit a follow-up PR to fix these? Preferably in the Miri repo.

Reply:
(None of these files had conflicts during rebase...)